src/DynamoPackages/PackageLoader.cs

```diff
@@ -85,7 +86,28 @@ public string DefaultPackagesDirectory
             get { return defaultPackagesDirectoryIndex != -1 ? packagesDirectories[defaultPackagesDirectoryIndex] : null; }
         }
 
-        private int defaultPackagesDirectoryIndex = -1;
+        private static string TransformPath(string root, string userDataFolder, string extension)
```

Comment:
could we add a summary for this even though it's internal? I'd like to know what type of transformation it's doing without reading the entire function.